File: AzureStorageSDK/PHP/UploadAzureStorage.php
<?php
  require_once "azure-sdk-for-php/WindowsAzure/WindowsAzure.php";
  require_once "vendor/autoload.php"; 
  
  use WindowsAzure\Common\ServicesBuilder;
  use WindowsAzure\Common\ServiceException;
    
  $mycontainer = "images";
  $myblob = "php.jpg";
  $myfile = "c:\\demo\\AzureStorageSDK\\AzureStorageSDKDemo.jpg";
  $connectionString = "DefaultEndpointsProtocol=https;AccountName=dev018storage;AccountKey=q4dLJhW3zLw8hmdjL88w0vc2KHrbjMJlXumFB0eZVpBoeks1MKlZh7+pW36fmlWmBMx+bzwvWcYMU8p4MGhbcg==";

   try {
    // Create blob REST proxy.
    $blobRestProxy = ServicesBuilder::getInstance()->createBlobService($connectionString);

    //Upload blob
    $blobRestProxy->createBlockBlob($mycontainer, $myblob, fopen($myfile, "r"));
    fclose($myfile);
    
    print "\n\"".$myblob."\" is Uploaded by PHP!\n";
  } catch(ServiceException $e){
    $code = $e->getCode();
    $error_message = $e->getMessage();
    echo $code.": ".$error_message;
  }
?>

File: AzureStorageSDK/Golang/UploadAzureStorage.go
package main

import(
  "fmt"
  "os"
  "github.com/loldesign/azure"
)

func main() {
	var mycontainer = "images"
  var myblob = "golang.jpg"
  var myfile = "c:\\demo\\AzureStorageSDK\\AzureStorageSDKDemo.jpg"
  
  var accountName = "dev018storage"
  var secret = "q4dLJhW3zLw8hmdjL88w0vc2KHrbjMJlXumFB0eZVpBoeks1MKlZh7+pW36fmlWmBMx+bzwvWcYMU8p4MGhbcg=="

  blob := azure.New(accountName, secret)
  file, err := os.Open(myfile)

  if err != nil {
    fmt.Println(err)
  }

  res, err := blob.FileUpload(mycontainer, myblob, file)

  if err != nil {
    fmt.Println(err)
  }

  fmt.Printf("\n%s\n\"%s\" Uploaded by Golang\n", res.Status, myblob)
}